package.json

```diff
@@ -23,39 +26,77 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/components-pkg.git"
+  },
+  "homepage": "https://github.com/smg-automotive/components-pkg#readme",
+  "peerDependencies": {
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
   "devDependencies": {
+    "@babel/core": "7.17.10",
+    "@react-theming/storybook-addon": "1.1.5",
     "@rollup/plugin-commonjs": "22.0.0",
     "@rollup/plugin-node-resolve": "13.3.0",
     "@rollup/plugin-typescript": "8.3.2",
+    "@storybook/addon-actions": "6.5.0-beta.8",
+    "@storybook/addon-controls": "6.5.0-beta.8",
+    "@storybook/addon-docs": "6.5.0-beta.8",
+    "@storybook/addon-essentials": "6.5.0-beta.8",
+    "@storybook/addon-interactions": "6.5.0-beta.8",
+    "@storybook/addon-links": "6.5.0-beta.8",
+    "@storybook/addon-postcss": "2.0.0",
+    "@storybook/builder-webpack5": "6.5.0-beta.8",
+    "@storybook/jest": "0.0.10",
+    "@storybook/manager-webpack5": "6.5.0-beta.8",
+    "@storybook/mdx2-csf": "0.0.3",
+    "@storybook/react": "6.5.0-beta.8",
+    "@storybook/testing-library": "0.0.11",
     "@testing-library/jest-dom": "5.16.4",
-    "@testing-library/react": "13.2.0",
+    "@testing-library/react": "12.1.5",
     "@testing-library/user-event": "14.1.1",
+    "@types/estree": "0.0.51",
     "@types/jest": "27.5.0",
-    "@typescript-eslint/eslint-plugin": "5.22.0",
-    "@typescript-eslint/parser": "5.22.0",
-    "eslint": "8.14.0",
+    "@types/react": "17.0.45",
+    "@typescript-eslint/eslint-plugin": "5.23.0",
+    "@typescript-eslint/parser": "5.23.0",
+    "autoprefixer": "10.4.7",
+    "babel-loader": "8.2.5",
+    "eslint": "8.15.0",
     "eslint-config-airbnb": "19.0.4",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jest": "26.1.5",
     "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-mdx": "1.17.0",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.5.0",
-    "eslint-plugin-testing-library": "5.3.1",
-    "jest": "28.0.3",
-    "jest-environment-jsdom": "28.0.2",
+    "eslint-plugin-storybook": "0.5.11",
+    "eslint-plugin-testing-library": "5.4.0",
+    "jest": "28.1.0",
+    "jest-environment-jsdom": "28.1.0",
+    "postcss-flexbugs-fixes": "5.0.2",
     "prettier": "2.6.2",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
```

Comment:
Chakra v1 doesn't support react 18, we get a lot of type errors because they changed type for `children` etc.
We can track the progress here: https://github.com/chakra-ui/chakra-ui/issues/5855